Main/src/main/java/com/main/acad/annotation/AnnotationForControllers.java

```diff
@@ -0,0 +1,12 @@
+package com.main.acad.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(value = ElementType.TYPE)
+@Retention(value = RetentionPolicy.RUNTIME)
+public @interface AnnotationForControllers {
```

Comment:
Please, rename it more creatively :), for example Path or Mapping or smth like this